client-js/app/elements/labrad-registry.html

```diff
@@ -97,6 +98,10 @@
     dir-item {
       white-space: nowrap;
     }
+    .mono {
+      font-family: 'roboto mono', monospace;
```

Comment:
That's the idea. I'm actually not sure how to best test this, since my machines probably already have the fonts installed.